### ISSUE TITLE
IntersectionTypeAnnotation needs to be implemented

### ISSUE THREAD

**Issue by k2snowman69** [snowcoders/sortier#20]
Input:
```
type A = "B" | "A";
type B = A & {
    Other: "C"
}
type C = {
    Other: "C"
} & A;
```

Output:
```
type A = "A" | "B";
type B = A & {
    Other: "C"
}
type C = A & {
    Other: "C"
};
```

**Comment by k2snowman69** [comment]
Will be resolved in >=2.1.6

**Comment by k2snowman69** [comment]
And published!
